packages/web/config/user-analytics-v2.ts

```diff
@@ -129,5 +132,7 @@ export const EventName = {
     depositAssetCompleted: "Deposit asset: Deposit completed",
     withdrawAssetStarted: "Withdraw asset: Withdraw started",
     withdrawAssetCompleted: "Withdraw asset: Withdraw completed",
+    buyOsmoStarted: "Assets: Buy Osmo started",
+    buyOsmoCompleted: "Assets: Buy Osmo completed",
```

Comment:
I see buy osmo events in amplitude, but just buy osmo clicked for the assets page